WordPress/Classes/ViewRelated/Voice/VoiceToContentView.swift

```diff
@@ -163,13 +148,13 @@ private struct RecordButton: View {
             Button(action: isRecording ? viewModel.buttonDoneRecordingTapped : viewModel.buttonRecordTapped) {
                 if #available(iOS 17.0, *) {
                     icon
-                        .contentTransition(.symbolEffect(.replace, options: .speed(4)))
+                        .contentTransition(.symbolEffect(.replace, options: .speed(5)))
                 } else {
                     icon
                 }
             }
 
-            Text(Strings.done)
+            Text(isRecording ? Strings.done : Strings.beginRecording)
```

Comment:
For accessibility, this Text could be combined with a button, to indicate that tapping the button begins the recording. Or the button could have "Begin recording" / "Stop recording" accessibility label.

Reply:
Great point. I included it in the button and also added an accessibility idenfieir for a "close" button.